src/core.js

```diff
@@ -617,15 +617,19 @@ JSONEditor.prototype = {
       }
       var ref = fetchUrl + refObj.$ref
       if (!this.refs[ref]) ref = fetchUrl + decodeURIComponent(refObj.$ref)
+      if (!this.refs[ref]) { // if reference not found
+        console.warn("reference:'" + ref + "' not found!")
+        break
+      }
```

Comment:
warning if the reference not found

Reply:
@schmunk42 @germanbisurgi When an error like this occur, should we use "console", "alert" or "throw new Error" ?
I'm not clear on that myself, but I think "throw new Error" would be the best approach.

Reply:
if breaks throw error. if not then warn

Reply:
The `exandRefs` should works alright even if the reference is not found, if a template reference is used , for example the reference in `describedBy` editors. So it think warning will be better.